draft-ounsworth-csr-attestation.md

```diff
@@ -363,6 +374,62 @@ Columns:
 
 # Security Considerations
 
+A PKCS#10 or CRMF Certification Request message consists of a
+distinguished name, a public key, and optionally a set of attributes,
+collectively signed by the entity requesting certification.
+The private key used to sign the CSR MUST
+be different from the Attesting Environment key used to sign Evidence about the Target Environment, which controls the private key used to sign the CSR. To
```

Comment:
Are there cases in DICE where it could end up being the same key material?

Reply:
Yes, you could use the same key -- not just in DICE. From a key separation point of view that's not a good idea.